app/assets/stylesheets/player.scss

```diff
@@ -209,7 +206,7 @@ body {
     text-align: center;
   }
 
-  .search-boxes, #jam-name, input, select {
+  #jam-name, input, select {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line